sdks/python/container/boot.go

```diff
@@ -371,6 +371,16 @@ func installSetupPackages(files []string, workDir string, requirementsFiles []st
 		log.Printf("Failed to setup acceptable wheel specs, leave it as empty: %v", err)
 	}
 
+	pkgName := "apache-beam"
+	isSdkInstalled, err := isPackageInstalled(pkgName)
+	if err != nil {
+		return fmt.Errorf("failed to check if Apache Beam %s is installed: %v", pkgName, err)
+	}
+
+	if !isSdkInstalled {
+		log.Printf("Apache Beam is not installed on the custom container. Please make sure Apache Beam is installed in the custom container. Look at the docummentation https://beam.apache.org/documentation/runtime/environments/ on custom containers for more details.")
```

Comment:
wording suggestion: "Apache Beam is not installed in the runtime environment. If you use a custom container image, you must install apache-beam package in the custom image using same version of Beam as in the pipeline submission environment. For more information, see: the https://beam.apache.org/documentation/runtime/environments/ page."

Reply:
Changed wording